package/features/steps/cli_steps.py

```diff
@@ -91,6 +92,19 @@ def exec_kedro_target_checked(context, command):
         print(res.stderr)
         assert False
 
+    # Wait for subprocess completion since on Windows it takes some time
+    # to install dependencies in a separate console
+    if "install" in cmd:
```

Comment:
We have `wait_for` util function to do this kind of checking.

Reply:
@921kiyo It doesn't fit a bit because it validates a result returned by `func` but I need to access the result attribute (exit code) to make sure that function was executed successfully.